Improve account abstraction docs (#87)
vechain/vechain-docs: core-concepts/account-abstraction/bundler.md

## Overview

> A Bundler plays a crucial role in the account abstraction ecosystem by continuously monitoring a public mempool for UserOperations created by user. Its primary responsibility is to collect these UserOperations, bundle them together, and pass them to the EntryPoint contract for execution.

Reviewer: It's correct, I'm reformulating the first part to avoid people thinking that the mempool is creted by the user.
```suggestion
A Bundler plays a crucial role in the account abstraction architecture. Its primary responsibility is to collect these UserOperations from a public mempool, bundle them together, and pass them to the EntryPoint contract for execution.
```

> The `EntryPoint contract` serves as a "global trusted singleton," meaning it is a single, universally trusted smart contract within the blockchain ecosystem. This contract handles incoming transactions, validates them, and manages the execution of UserOperations. The Bundler acts as the intermediary between users and the EntryPoint contract, ensuring UserOperations are properly packaged and processed.

Reviewer: ```suggestion
The `EntryPoint contract` serves as a "global trusted singleton," meaning it is a single, universally trusted smart contract within the blockchain ecosystem.  In reality it's not like that, in a public blockchain anyone can create and promote its EP, do your own research and trust the EP you can find on trusted websites only.
This contract handles incoming transactions, validates them, and manages the execution of UserOperations. The Bundler acts as the intermediary between users and the EntryPoint contract, ensuring UserOperations are properly packaged and processed.
```
